### PR TITLE
Update the README.md

### PR DESCRIPTION
**Pull request by Arcanemagus**
Fix the configuration path and link to the official list of linters.

Fixes https://github.com/AtomLinter/linter-php/issues/52.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,17 +13,9 @@ Linter package must be installed in order to use this plugin. If Linter is not i
...
-- [linter-jshint](https://atom.io/packages/linter-jshint) - Linter plugin for JavaScript, using jshint.
-- [linter-scss-lint](https://atom.io/packages/linter-scss-lint) - Sass Linter plugin for SCSS, using scss-lint.
-- [linter-coffeelint](https://atom.io/packages/linter-coffeelint) Linter plugin for CoffeeScript, using coffeelint.
-- [linter-csslint](https://atom.io/packages/linter-csslint) Linter plugin for CSS, using csslint.
-- [linter-rubocop](https://atom.io/packages/linter-rubocop) - Linter plugin for Ruby, using rubocop.
-- [linter-tslint](https://atom.io/packages/linter-tslint) Linter plugin for TypeScript, using tslint.
 
-## Donation
-[![Share the love!](https://chewbacco-stuff.s3.amazonaws.com/donate.png)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=KXUYS4ARNHCN8)
+## Other linters
```
Comment (steelbrain) [inline README.md]: How about you remove this part completely?

Comment (Arcanemagus) [inline README.md]: A little bit of scope creep in adding a `coffeelint.json` as well, but fixed.

**Comment by steelbrain** [comment]
Beautiful!

File: coffeelint.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "max_line_length": {
+		"value": 120,
```
Comment (steelbrain) [inline coffeelint.json]: I think you might have an indent problem here

Comment (Arcanemagus) [inline coffeelint.json]: Silly Atom keeps switching to tabs on spaces files...
